### DISCUSSION TITLE
How the clusters are identified?

### DISCUSSION THREAD

**Discussion by huizizhang949** [theislab/scvelo#881] [Q&A] [answered]
Hi,

I have a dataset with 10 defined clusters in the beginning. But after running the dynamic model and velocity estimation, the velocity embedding shows 12 clusters. Is there any step in the middle to identify clusters? How?

Thank you!

**Comment by WeilerP** [comment, marked as answer]
No, there isn't. My best guess/suggestion is to check that you are coloring the embedding according to the correct column in `adata.obs`, _i.e._, the `color` argument is set correctly.

**Reply by huizizhang949** [reply]
Thank you! I notice that my embedding 'title' is louvain which seems to be new column of labels added to the data. Are they clusters? How can I add my clusters 'adata.obs.Clusters' as labels?

**Reply by WeilerP** [reply]
You must have run Louvain clustering at some point. To color the embedding by according to the column `'Clusters'`, you need to pass `color='Clusters'`. Please have a look at the [docs](https://scvelo.readthedocs.io/scvelo.pl.velocity_embedding_stream/#scvelo.pl.velocity_embedding_stream).
